lib/bloc/auth/bloc.dart

```diff
@@ -21,7 +21,7 @@ class AuthBloc extends Bloc<AuthEvents, AuthStates> {
     if (event is AppStarted) {
       final bool hasToken = await auth.hasToken();
 
-      if (hasToken) {
+      if (!hasToken) {
```

Comment:
![](https://media.giphy.com/media/ChaKoMFCkuNPi/giphy.gif)

Reply:
Logically it's still wrong but oh well.